satpy/tests/reader_tests/test_viirs_gran.py

```diff
@@ -0,0 +1,242 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# Copyright (c) 2019 Satpy developers
+
+# Author(s):
+
+#
+#   Barry Baker @bbakernoaa GitHub
+#   David Hoese <david.hoese@ssec.wisc.edu>
+#   Daniel Hueholt <daniel.hueholt@noaa.gov>
+#   Tommy Jasmin <tommy.jasmin@ssec.wisc.edu>
```

Comment:
Please merge with current master and copy license header from other files. I recently went through and standardized them. Please double check that Barry and you are in the `AUTHORS.md` list. Tommy will be added in his TROPOMI reader PR. I fixed a lot of the names in the authors list so you should only have to worry about you and Barry. It is alphabetical by last name. Feel free to stop by my desk if you need help.